share/lib/python/neuron/rxd/node.py

```diff
@@ -350,8 +359,11 @@ def include_flux(self, *args, **kwargs):
                     f = float(args[0])
                     source = f
                     success = True
-                except:
-                    pass
+                except TypeError:
+                    arith = rxdmath._ensure_arithmeticed(args[0])
+                    source = lambda: eval_arith_flux(arith, self.region, self)
+                    scale = 1 / self.volume
+                    success = True
             if not success:
                 raise RxDException("unsupported flux form")
         _node_fluxes["index"].append(self._index)
```

Comment:
possibly a problem for a new issue, but what happens if we run a test after a test using node fluxes? Do they get properly garbage collected?